src/App/App.tsx

```diff
@@ -86,7 +86,7 @@ export default class App extends Component<{}, AppStateInterface> {
       this.handleLogout();
     } else {
       this.setState({
-        user: { username, token },
```

Comment:
:-) we need the token otherwise you cannot login.

Reply:
sure, but `token` is not defined in `AppStateInterface`

Reply:
it gets written to `storage.setItem('token', token);` instead